src/Microsoft.ML.CpuMath/AvxIntrinsics.cs

```diff
@@ -417,6 +417,8 @@ public static unsafe void AddScalarU(float scalar, Span<float> dst)
             {
                 float* pDstEnd = pdst + dst.Length;
                 float* pDstCurrent = pdst;
+                int destinationEnd = pDstEnd - 4;
+
```

Comment:
nit: stray newline here

Reply:
This should be updated to remove the extra new line.